rca/people/models.py

```diff
@@ -872,6 +880,44 @@ def student_related_links(self):
             for item in self.relatedlinks.all()
         ]
 
+    def save(self, *args, **kwargs):
+        """On saving the student page, make sure the student_user_account
+        has a group created with the necessary permissions
+        """
+        super(StudentPage, self).save()
+        if self.student_user_image_collection and self.student_user_image_collection:
+            # Create a specific group for this student so they have edit access to their page
+            # and their image collection
+            specific_student_group, created = Group.objects.get_or_create(
```

Comment:
We should check for the `GroupPagePermission` existing first before creating anything.